src/applications/ivc-champva/10-7959C/config/submitTransformer.js

```diff
@@ -1,16 +1,72 @@
 import { transformForSubmit as formsSystemTransformForSubmit } from 'platform/forms-system/src/js/helpers';
 
+/**
+ * Convert a list of strings into an object with boolean
+ * keys indicating the presence or absence of a key. E.g.,
+ * if given list ['prop1', 'prop2'], return would be:
+ * {is_prop1: true, is_prop2: true}
+ * @param {array} listOfStr List of health insurance types (e.g., ['ppo', 'medigap'])
+ * @returns Object of boolean properties based on list of strings provided, e.g.,:
+ * {is_prop1: true, is_prop2: true}
+ */
+function listOfStrToBools(listOfStr) {
+  let retVal = {};
+  if (listOfStr !== undefined) {
+    listOfStr
+      .map(key => {
+        return { [`is_${key}`]: true };
+      })
+      .forEach(obj => {
+        retVal = { ...retVal, ...obj };
+      });
+  }
+  return retVal;
+}
+
+/**
+ * This function recursively finds all object properties with a value of "yes" or "no"
+ * and converts those values in-place to booleans. E.g., if provided obj = {prop1: "yes"},
+ * will modify obj so that it === {prop1: true}.
+ * @param {object} data Object we're searching for string values to convert to bools
+ * @param {number} depth Current recursion depth (used to prevent runaway)
+ * @param {number} maxDepth Maximum recursion depth
+ * @returns undefined, operates on obj in place
+ */
+function yesNoToBool(data, depth = 0, maxDepth = 5) {
+  const obj = data;
+  // Prevent infinite recursion if we somehow have self-referencing object
+  if (depth > maxDepth) return;
+  // Check all keys on current object and recurse if necessary
+  Object.keys(obj).forEach(k => {
+    if (obj[k] === 'yes') obj[k] = true;
+    else if (obj[k] === 'no') obj[k] = false;
+    else if (typeof obj[k] === 'object')
+      yesNoToBool(obj[k], depth + 1, maxDepth);
+  });
+}
+
```

Comment:
This will similarly make it easy for the backend to check checkboxes on the PDF, as it will have real booleans now rather than strings of "yes" and "no". This was the simplest way to bridge the gap without having to change the values put out by all radio fields in the form.